### PR TITLE
Feature/add firebase authorization

### PR DESCRIPTION
**Pull request by agatty-chacrasoftware**

### CODE REVIEW

File: src/helpers/profilePictureService.ts [hunk 1]
```diff
@@ -1,6 +1,38 @@
 import { PrismaClient } from ".prisma/client";
 import { getEmployeeById } from "./employeeService";
 const prisma = new PrismaClient();
+import dotenv from "dotenv";
+dotenv.config();
```
Comment (iskandar-chacrasoftware) [inline src/helpers/profilePictureService.ts]: dotenv.config should be called once in your index file

File: prisma/schema.prisma [hunk 2]
```diff
@@ -18,7 +18,7 @@ model Employee {
   department   Department @relation(fields: [departmentId], references: [departmentId])
   departmentId Int
   projects ProjectAssignment[]
-  profilePictureUrl EmployeeProfilePicture?
+  image EmployeeProfilePicture?
 }
 
 model EmployeeProfilePicture {
```
Comment (iskandar-chacrasoftware) [inline prisma/schema.prisma]: Why is this a separate model?

File: src/helpers/profilePictureService.ts [hunk 3]
```diff
@@ -1,6 +1,38 @@
...
+const cloudinary = require("cloudinary").v2;
+
+export const uploadImageToCloudinary = async (createReadStream) => {
+	cloudinary.config({
+		cloud_name: process.env.CLOUDINARY_CLOUD_NAME,
+		api_key: process.env.CLOUDINARY_API_KEY,
+		api_secret: process.env.CLOUDINARY_API_SECRET,
+	});
+
+	const { secure_url: profilePictureUrl } = await new Promise(
```
Comment (iskandar-chacrasoftware) [inline src/helpers/profilePictureService.ts:4]: This code seems a bit complex for what it's trying to do. Is this copied from somewhere?

File: prisma/schema.prisma (hunk 2 above)
Comment (agatty-chacrasoftware) [reply prisma/schema.prisma]: This was the initial Idea. Was not sure on how to implement with employee. So kept it in this way.

Comment (agatty-chacrasoftware) [reply prisma/schema.prisma]: I can change this and implement with employee now.

File: src/helpers/profilePictureService.ts (hunk 3 above)
Comment (agatty-chacrasoftware) [reply src/helpers/profilePictureService.ts:4]: Yes. I had checked one post. It seemed simpler.

File: src/helpers/profilePictureService.ts (hunk 1 above)
Comment (agatty-chacrasoftware) [reply src/helpers/profilePictureService.ts]: Changes for the above comment pushed.

File: prisma/schema.prisma (hunk 2 above)
Comment (agatty-chacrasoftware) [reply prisma/schema.prisma]: Implemented within employee.

File: src/helpers/profilePictureService.ts (hunk 3 above)
Comment (agatty-chacrasoftware) [reply src/helpers/profilePictureService.ts:4]: Changes have been pushed in new commit

**Pull request merged**